ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1864,3 +1864,24 @@ def test_paddle_diff(
         prepend=prepend[0],
         append=append[0],
     )
+
+
+def test_paddle_increment(
+    *,
+    dtype_and_input_and_label,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+):
+    input_dtypes, input_and_label = dtype_and_input_and_label
+    input, label = input_and_label
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        input=input,
+        label=label,
```

Comment:
The input parameters are `x` and `value`, not `input` and `label`. It seems to me as if you copied this test from somewhere but forgot to modify it for your own use case :)

Reply:
Yes indeed sorry for that